elixir/apps/domain/lib/domain/auth/adapters/openid_connect.ex

```diff
@@ -134,6 +135,7 @@ defmodule Domain.Auth.Adapters.OpenIDConnect do
           # if an email was used in provider identifier and it's replaced by sub claim
           # later, we want to use the ID from sub claim as provider_identifier
           |> Ecto.Changeset.put_change(:provider_identifier, provider_identifier)
+          |> Ecto.Changeset.put_change(:email, email)
```

Comment:
Just so I'm following, this comment is still relevant, right?

If the admin enters an email for the provider identifier when creating an identity, we'll still replace it with the provider_identifier upon first sign in?

Reply:
Yeah, that comment is still correct/valid and your understanding is correct.  We will update the `provider_identifier` with the `sub` on first sign in